pallets/pallet-deposit-storage/src/fungible/tests/burn.rs

```diff
@@ -0,0 +1,100 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2024 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use frame_support::traits::{
+	fungible::MutateHold,
+	tokens::{Fortitude, Precision},
+};
+use kilt_support::Deposit;
+use sp_runtime::AccountId32;
+
+use crate::{
+	deposit::DepositEntry,
+	fungible::{
+		tests::mock::{ExtBuilder, TestRuntime, TestRuntimeHoldReason, OWNER},
+		PalletDepositStorageReason,
+	},
+	Pallet, SystemDeposits,
+};
+
+#[test]
+fn burn_held() {
+	ExtBuilder::default()
+		.with_balances(vec![(OWNER, 100_000)])
+		.build()
+		.execute_with(|| {
+			let reason = PalletDepositStorageReason::default();
+
+			<Pallet<TestRuntime> as MutateHold<AccountId32>>::hold(&reason, &OWNER, 10)
```

Comment:
Using `StorageDepositPallet` would look simpler

Reply:
yes but its a minor thing

Reply:
I think I'll skip this, as we have both approaches across our codebase, so I think it's not too big of a deal. I personally prefer the current approach, which, once again, is more Rusty instead of relying on the result of the `construct_runtime` macro.